### PR TITLE
🔥[HOTFIX]: Frontend Bug Fixes

### PR DESCRIPTION
**Pull request by tholulomo**
- Fixed search autosuggest disabling at keyboard enter
- Add snack bar implementation for search failures
- Fix UI scrollbar bug

### CODE REVIEW

**Review by aswallace — CHANGES_REQUESTED** [review]
When I pulled these changes and tried it out locally, the autosuggest menu is still staying open when I search. I also left a comment about the snackbar infinite duration

**Comment by aswallace** [comment]
I tried this out some more, and it turns out that the autosuggest goes away if I wait for it to appear and then hit enter. But since it doesn't show up right away, I normally hit enter before it renders, so that by the time it re-navigates to the search result, the menu only then renders and then gets stuck there. (I know this isn't the best explanation, let me know if I can clarify)

File: app/src/components/Snackbar.vue [hunk 1]
```diff
@@ -1,11 +1,9 @@
 <template>
   <div>
-    <md-snackbar :md-position="position" :md-active.sync="show">
+    <md-snackbar :md-position="position" :md-active.sync="show" :md-duration="isInfinity">
```
[the remark this answers is not in the document]
Comment (tholulomo) [reply app/src/components/Snackbar.vue:3]: The intent for adding infinity is to ensure that persistency while making sure the user is aware of the error. I noticed without that, it display the error and it's gone before you notice. Also I am converting it to ternary in my current branch to accommodate duration. Later, we can add a listener to dismiss if any other button is clicked other than the action button

**Comment by tholulomo** [comment]
> I tried out the explorer search some more, and it turns out that the autosuggest goes away if I wait for it to appear and then hit enter. But since it doesn't show up right away, I normally hit enter before it renders, so that by the time it re-navigates to the search result, the menu only then renders and then gets stuck there. (I know this isn't the best explanation, let me know if I can clarify)

The browser sets that input field out of focus as soon as you press enter and I can notice a latency in respone.

**Comment by tholulomo** [comment]
This PR is now merged into #308

**Pull request closed without merging**